apps/central-scan/frontend/src/components/export_results_modal.tsx

```diff
@@ -156,13 +156,10 @@ export function ExportResultsModal({
         <Modal
           title={mode === 'cvrs' ? 'Save CVRs' : 'Save Backup'}
           content={
-            <React.Fragment>
-              <UsbImage />
```

Comment:
We should make the same change to remove these usb images elsewhere (e.g. VxAdmin)

Reply:
It's comin' up.